try:
 x=10
 exit()

except:
 print("Some problem")